### PR TITLE
Rick and Morty WebApp

### PR DESCRIPTION
**Pull request by SimonGrchevski**
Me and my team partner, Moise, were working on the JS capstone project and this is what we implemented :
- The home page content is dynamically added.
- Call the API to get  The Rick And Morty characters and the likes count.
- Add a feature to display a popup window when the user clicks the comment button
- Add the details of the character to the popup window and a feature to comment
- Add the personalized styles to the site

This is the video that I and my partner recorded for this project: 
- [Video Link](https://drive.google.com/file/d/1ZuysKZjdA84rTHmPnzlqc8lekO4IIGDY/view?usp=sharing)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      </ul>
+    </div>
+  </main>
+
+  <footer id="footer">
+    <div class="bg"></div>
+    <div class="main-footer-container flex">
+      <p>Created By microverse under CC lisense</p>
+    </div>
+  </footer>
```
Comment (carlos-ssh) [inline src/index.html:45]: - Fix your footer position.
Screen Shot 2021-08-05 at 07 32 09

File: src/api.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+export const post = async (url, endPoint, body) => {
+  const result = await fetch(`${url}${endPoint}`, {
+    method: 'POST',
+    body,
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  });
+  return result;
```
Comment (carlos-ssh) [inline src/api.js:16]: - Solve errors in your with your API:
Screen Shot 2021-08-05 at 07 33 58

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/api.js (hunk 2 above)
Comment (moise10r) [reply src/api.js:16]: On our end, access to fetch at Rick and Morty Api is not blocked by the core policy,
Screenshot from 2021-08-05 15-39-00

However, we added the Access-control-origin on the post request.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+## Live demo
+[See Project](https://simongrchevski.github.io/js-capstone/)
+
+## Built With
+
+- Javascript
+- CSS
+- Html
+
```
[the remark this answers is not in the document]
Comment (moise10r) [reply README.md:24]: We added the link to our video in the description of the pull request.

**Review by DeliceLydia — APPROVED** [review]
# Congratulations 🎉 🎉 

## Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**